### PR TITLE
@broskoski => Art links for onboarding, closes #9

### LINKED ISSUE #9: Art-world domain knowledge information
Links to blogs, etc.

### PR DESCRIPTION
**Pull request by bhoggard**

### CODE REVIEW

**Comment by broskoski** [comment]
Art publication suggestions:
[032c](http://032c.com/)
[Frieze](http://www.frieze.com/magazine/)
[Mousse](http://moussemagazine.it/)
[Contemporary Art Daily](http://www.contemporaryartdaily.com/)

**Comment by broskoski** [comment]
For art calendar links:
(zingrecsNY)[http://www.zingmagazine.com/drupal/zingrecsNY]

**Comment by dblock** [comment]
Can we maybe make a folder called `art` and move this into `art/resources.md` or something? This way we can start splitting this into `art/books.md` and what not in the future?

**Comment by dblock** [comment]
Oh and this list is awesome!

File: README.md [hunk 1]
```diff
@@ -47,6 +47,10 @@ In order to get things like Gravity set up, you’ll need the proper AWS credent
 
 We use [AWS](https://artsy.signin.aws.amazon.com/console) for storage (S3), hosting websites (OpsWorks), and analytics (Redshift), among other things.
 
+##### Art world notes
+
+Many of our engineers have art backgrounds or degrees, but of course not all of them. [Here](art.md). are some useful links if you want to read more about the art world, or keep up with the latest news.
```
Comment (orta) [inline README.md]: This renders a bit weird

screen shot 2015-06-04 at 3 41 29 pm

**Comment by bhoggard** [comment]
@broskoski I will add the other three, but 032c seems more fashion-y than the other sites, doesn't it?

**Comment by broskoski** [comment]
Yeah, that one felt like a bit of a stretch :)

**Comment by dzucconi** [comment]
This makes me wonder: does Artsy have institutional subscriptions to anything?

**Pull request merged**